remoteworkitem/trackeritem_repository.go

```diff
@@ -5,6 +5,14 @@ import "github.com/jinzhu/gorm"
 // upload imports the items into database
 func upload(db *gorm.DB, tqID int, item map[string]string) error {
 	ti := TrackerItem{Item: item["content"], RemoteItemID: item["id"], BatchID: item["batch_id"], TrackerQueryID: uint64(tqID)}
-	err := db.Create(&ti).Error
+	db.Where("remote_item_id = ?", item["id"]).Find(&ti)
+	var err error
+	if ti.ID == 0 {
+		ti1 := TrackerItem{Item: item["content"], RemoteItemID: item["id"], BatchID: item["batch_id"], TrackerQueryID: uint64(tqID)}
+		err = db.Create(&ti1).Error
+	} else {
+		ti2 := TrackerItem{ID: ti.ID, Item: item["content"], RemoteItemID: item["id"], BatchID: item["batch_id"], TrackerQueryID: uint64(tqID)}
```

Comment:
I understand this need more analysis. Since we are only working with public issue tracker, we can ignore auth related issues for now. Yes, the batch ID also can be removed for now.

If you are fine with this. I can add separate issues to work on these problems and remove batch ID from the current implementation.